app/models/concerns/taggable.rb

```diff
@@ -0,0 +1,17 @@
+module Taggable
```

Comment:
Ah yes, we can probably get rid of the `cached_tags`. I didn't think about removing it because it was needed to have ES work with acts-as-taggable-on caching, but since I'm not using it, I can probably ditch it.

It gets indexed twice, but only the stuff that has changed. The first time, it might be just the title, but the second pass will be for tags (and only when the record gets a new tag). At least, we are not sending the whole Resource (and its content) to be re-indexed two times :p